### DISCUSSION TITLE
How to minify tailwind in page css

### DISCUSSION THREAD

**Discussion by vazgen** [nuxt/nuxt#9465] [Questions (Nuxt 2)]
Hello, I use Nuxt 2.15.7 and I added tailwind css to it. Everythings work fine. Unused css being removed from the page. But the problem is that css is not minified (please check an example page https://www.w3docs.com/course). I can extract css from the file and minify it but I would like to keep it in the file but minified. 

Thank you!
<img width="843" alt="a" src="https://user-images.githubusercontent.com/830160/122845298-732bf780-d314-11eb-821b-64f974201742.png">

**Comment by kissu** [comment]
Do you have a reproduction of this? Like a codesandbox. 😄

**Reply by vazgen** [reply]
No, I do not. Where can I setup it?

**Reply by kissu** [reply]
On codesandbox itself: https://codesandbox.io/
